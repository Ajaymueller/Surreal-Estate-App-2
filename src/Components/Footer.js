import React from 'react';
import "../styles/Footer.css";

const Footer = () => {
    return (
        <div className="footer">
            <ul className="footer_content">
                <li className="footer__item" data-testid="footer-id">
                    Aidan Mueller, 2020</li>
            </ul>
        </div>
    );
};

export default Footer;