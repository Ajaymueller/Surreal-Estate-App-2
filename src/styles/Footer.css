.footer {
    display: flex;
    justify-content: center;

    margin: 0px;
    padding-left: 20px;
    
    background-color: var(--tertiary-dark-color);
    border-top: 1px solid var(--tertiary-dark-color);

}

.footer_content {
    list-style: none;
    padding-left: 5px;
}

.footer li {
    font-size: larger;
    font-weight: bold;
}