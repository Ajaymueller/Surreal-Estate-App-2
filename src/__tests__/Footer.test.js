import React from 'react';
import { render, fireEvent, getAllByTestId } from '@testing-library/react';
import Footer from '../Components/Footer';
import { MemoryRouter } from 'react-router-dom';
import "@testing-library/jest-dom/extend-expect";

describe("footer", () => {
    it("component renders correctly", () => {
    const { asFragment } = render (
    <MemoryRouter><Footer /></MemoryRouter> )
    expect(asFragment).toMatchSnapshot();
    });
    it("renders li with name and year as content", () => {
    const { getByTestId } = render (
    <MemoryRouter><Footer /></MemoryRouter> )
    const li = getByTestId("footer-id");
    expect(li).toBeInTheDocument();
    expect(li.innerHTML).toBe("Aidan Mueller, 2020");
    });
});