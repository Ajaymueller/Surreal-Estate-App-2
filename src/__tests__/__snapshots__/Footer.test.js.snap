exports[`footer component renders correctly 1`] = `[Function]`;
